docs/openapi.yaml

```diff
@@ -897,6 +897,15 @@ components:
             resource:
               type: string
               example: '/programs/DEV/projects/test'
+              description: >-
+                The full path for a resource a user is trying to access. Either
+                this or the tag is required.
+            tag:
```

Comment:
minor update to remove this as separate field and add into resource description how you can specify a tag too